authentication-ldap/src/main/java/com/synopsys/integration/alert/authentication/ldap/action/LDAPTestAction.java

```diff
@@ -51,6 +57,18 @@ protected ConfigurationTestResult testConfigModelContent(LDAPConfigTestModel lda
         String password = ldapConfigTestModel.getTestLDAPPassword();
         Authentication pendingAuthentication = new UsernamePasswordAuthenticationToken(username, password);
 
+        LDAPConfigModel ldapConfigModel = ldapConfigTestModel.getLdapConfigModel();
+        // If the Password is NOT set and IsManagerPasswordSet is true in the current model, get the PW from the DB
+        if (StringUtils.isBlank(ldapConfigModel.getManagerPassword().orElse("")) && Boolean.TRUE.equals(ldapConfigModel.getIsManagerPasswordSet())) {
+            ldapConfigAccessor.getConfiguration()
+                .flatMap(LDAPConfigModel::getManagerPassword)
+                .ifPresent(ldapConfigModel::setManagerPassword);
+        }
```

Comment:
If using a previously saved LDAP configuration, the PW will not be present in the UI. If that is the case, get the PW from the DB